app/src/main/java/ch/epfl/balelecbud/PointOfInterestActivity.java

```diff
@@ -6,19 +6,25 @@
 import androidx.recyclerview.widget.RecyclerView;
 import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
 
+import com.google.android.material.navigation.NavigationView;
+
 import ch.epfl.balelecbud.pointOfInterest.PointOfInterest;
 import ch.epfl.balelecbud.pointOfInterest.PointOfInterestData;
 import ch.epfl.balelecbud.pointOfInterest.PointOfInterestHolder;
 import ch.epfl.balelecbud.util.views.RecyclerViewData;
 import ch.epfl.balelecbud.util.views.RefreshableRecyclerViewAdapter;
 
-public class PointOfInterestActivity extends BasicActivity{
+public class PointOfInterestActivity extends BasicActivity implements NavigationView.OnNavigationItemSelectedListener{
```

Comment:
`BasicActivity` already implements `NavigationView.OnNavigationItemSelectedListener`

Reply:
Yup, missed it during the refactoring